rackcity/views/power_views.py

```diff
@@ -555,8 +557,6 @@ def make_bcman_request(chassis, blade, power_command):
         result = fp.read().splitlines()[0]
         fp.close()
         os.remove("temp.txt")
-    else:
-        result = "temp.txt file does not exist"
```

Comment:
this is something i had pushed to dev directly (muahaha) while testing so i'm removing it now